streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/controller/PassportController.java

```diff
@@ -124,4 +116,24 @@ private Map<String, Object> generateUserInfo(JWTToken token, User user) {
         return userInfo;
     }
 
+    private Map<String, Object> login(String username, String password, User user) throws Exception {
+        if (user == null) {
+            return RestResponse.success().put("code", 0);
+        }
+
+        if (User.STATUS_LOCK.equals(user.getStatus())) {
+            return RestResponse.success().put("code", 1);
+        }
```

Comment:
> Hi @MonsterChenzhuo , thanks for your contribution. I have a question, please take a look in your free time, thanks~
> 
> From old code, the return value of this method should be the UserInfo.
> 
> But when the `user == null || User.STATUS_LOCK.equals(user.getStatus())`, the return value will be the `RestResponse`. It might be a bug.

+1

Reply:
@1996fanrui 
<img width="957" alt="图片" src="https://user-images.githubusercontent.com/60029759/194683538-860cf1c2-a0aa-4271-9f58-60b9c8a294f1.png">
Do you think it's okay to make changes like this?
This `RestResponse.fail("The user is currently locked", 1L)` second parameter needs to pass an error code, but I saw that the project does not currently have this error code list, so I filled in a random one.